### PR TITLE
Added documentation for the alert in the login page

### PR DESCRIPTION
**Pull request by mackdk**
# Description

PR uyuni-project/uyuni/pull/4224 adds an alert in the login page when the space on the server is running out. This PR adds in the documentation the reference to this new alert.

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.2 uyuni-project/uyuni-docs/pull/1209
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

uyuni-project/uyuni/pull/4224
SUSE/spacewalk/issues/14259

### CODE REVIEW

File: modules/administration/pages/space-management.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@ Running out of disk space can have a severe impact on the {productname} database
 You can modify which directories are monitored, and the warnings that are created.
 All settings are configured in the [path]``/etc/rhn/rhn.conf`` configuration file.
 
+When the available space in one of these monitored directories falls below a warning threshold, an email is sent to the configured email address and a message is shown at the top of the login page.
```
Comment (keichwa) [inline modules/administration/pages/space-management.adoc:10]:
```suggestion
When the available space in one of the monitored directories falls below a warning threshold, a message is sent to the configured email address and a notofication is shown at the top of the sign-in page.
```

File: modules/administration/pages/space-management.adoc [hunk 2]
```diff
@@ -68,3 +71,5 @@ You can disable it entirely with these commands:
 systemctl stop spacewalk-diskcheck.timer
 systemctl disable spacewalk-diskcheck.timer
 ----
+
+Doing this disables the periodic email alerts if the alert threshold is reached, but the warning message will still appear at the top of the login page.
```
Comment (keichwa) [inline modules/administration/pages/space-management.adoc:75]:
```suggestion
Disabling the [literal]``spacewalk-diskcheck.timer`` will stop periodic email alerts if the alert threshold is reached, but the warning notification will still appear at the top of the sign-in page.
```

**Review by keichwa — APPROVED** [review]
minor language proposals.

File: modules/administration/pages/space-management.adoc (hunk 1 above)
Comment (mackdk) [reply modules/administration/pages/space-management.adoc:10]: There is a typo ("notofication") in your suggested change. If you edit it, I will commit it together with the other one.

**Comment by keichwa** [comment]
@mackdk , do we also want to have it backported to 4.2?  If yes, I can do it when this one is merged.

**Comment by mcalmer** [comment]
> @mackdk , do we also want to have it backported to 4.2? If yes, I can do it when this one is merged.

Maybe yes. AFAIK features done in the second half of the year are normally backported to the last released version.

**Comment by keichwa** [comment]
maybe, documentation backport to 4.2 is not that important.

**Pull request merged**